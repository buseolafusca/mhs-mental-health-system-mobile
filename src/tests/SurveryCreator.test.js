import React from 'react'
import { shallow } from 'enzyme'
import Creator from '../layouts/SurveyCreator'

describe('Location page', () => {
  it('should render correctly in mode', () => {
    const component = shallow(<Creator/>)
    expect(component).toMatchSnapshot()
  })
})
