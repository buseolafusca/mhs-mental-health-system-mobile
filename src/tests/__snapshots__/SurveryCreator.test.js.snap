// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location page should render correctly in mode 1`] = `
<div
  id="surveyCreatorContainer"
/>
`;
